.github/workflows/pr_labeler.yml

```diff
@@ -1,12 +0,0 @@
-name: "Pull Request Labeler"
```

Comment:
This file wasnt deleted, it just changed a lot. See: .github/workflows/on-pr_pr-labeler.yml